packages/teams-js/src/public/media.ts

```diff
@@ -412,14 +412,17 @@ export namespace media {
      * @param mediaEvent indicates what the event that needs to be signaled to the host client
      * Optional; @param callback is used to send app if host client has successfully handled the notification event or not
      */
-    protected notifyEventToHost(mediaEvent: MediaControllerEvent, callback?: (err?: SdkError) => void): void {
+    protected notifyEventToHost(
+      mediaEvent: MediaControllerEvent,
+      callback?: (err?: SdkError) => void,
+    ): void | SdkError {
       ensureInitialized(FrameContexts.content, FrameContexts.task);
-      const err = isApiSupportedOnMobile(nonFullScreenVideoModeAPISupportVersion);
-      if (err) {
-        if (callback) {
-          callback(err);
-        }
-        return;
+      let err;
+      try {
+        isApiSupportedOnMobile(nonFullScreenVideoModeAPISupportVersion);
+      } catch (e) {
+        err = e;
+        return callback(err);
       }
```

Comment:
After the other changes I mentioned, you shouldn't need to catch here. We want this exception to bubble up and be thrown to the app developer because they are trying to call something on a platform where it isn't supported